### ISSUE TITLE
Gatsby New not working as expected

### ISSUE THREAD

**Issue by dumblole** [gatsbyjs/gatsby#8325]
## Description

Can't start a new gatsby project

### Steps to reproduce

https://next.gatsbyjs.org/docs/
```
gatsby new gatsby-site https://github.com/gatsbyjs/gatsby-starter-default#v2
```
cd into directory and `gatsby develop`
`npm install` to make sure all dependencies are installed.

### Expected result

I also didn't come to any of these errors before gatsby v2 came out but I'm not sure since I was introduced to gatsby the day before the update.
Smooth installation of all dependencies when running gatsby new.
And being able to do `gatsby develop`

### Actual result

```
E:\Computer-Stuff\Javascript\2018>gatsby new gatsby-site https://github.com/gatsbyjs/gatsby-starter-default#v2
info Creating new site from git: https://github.com/gatsbyjs/gatsby-starter-default.git
Cloning into 'gatsby-site'...
remote: Counting objects: 905, done.
Receiving objects:  68% (616/905)    0 (delta 0), pack-reused 905
Receiving objects: 100% (905/905), 922.96 KiB | 3.16 MiB/s, done.
Resolving deltas: 100% (521/521), done.
success Created starter directory layout
info Installing packages...
yarn install v1.9.4
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] Resolving packages...
info There appears to be trouble with your network connection. Retrying...
[2/4] Fetching packages...
[--------------------------------------------------------------------------------------------------------------] 0/1356(node:9360) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
info fsevents@1.2.4: The platform "win32" is incompatible with this module.
info "fsevents@1.2.4" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
[4/4] Building fresh packages...
error E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp: Command failed.
Exit code: 1
Command: (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)
Arguments:
Directory: E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp
Output:
info sharp Using cached C:\Users\Super\AppData\Roaming\npm-cache\_libvips\libvips-8.6.1-win32-x64.tar.gz
(node:9576) UnhandledPromiseRejectionWarning: ZlibError: zlib: unexpected end of file
    at Zlib.(anonymous function).onerror (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:131:21)
    at Unzip.write (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:255:31)
    at Unzip.flush (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:222:10)
    at Unzip.end (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:229:10)
    at Unpack.end (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\tar\lib\parse.js:415:21)
    at Yallist.pipes.forEach.p (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minipass\index.js:288:18)
    at Yallist.forEach (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\yallist\yallist.js:154:8)
    at ReadStream.emit (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minipass\index.js:285:18)
    at ReadStream.emit (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\fs-minipass\index.js:153:22)
    at ReadStream.[maybeEmitEnd] (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minipass\index.js:255:12)
(node:9576) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:9576) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
info sharp Creating E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\build\Release
info sharp Copying DLLs from E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\vendor\lib to E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\build\Release
prebuild-install WARN install The specified module could not be found.
prebuild-install WARN install \\?\E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\build\Release\sharp.node

E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "" rebuild )
gyp info it worked if it ends with ok
gyp info using node-gyp@3.8.0
gyp info using node@10.10.0 | win32 | x64
gyp ERR! configure error
gyp ERR! stack Error: Command failed: c:\users\super\appdata\local\programs\python\python36-32\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:289:12)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:962:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
gyp ERR! System Windows_NT 10.0.17134
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp
gyp ERR! node -v v10.10.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed: yarnpkg


  Error: Command failed: yarnpkg

  - index.js:236 Promise.all.then.arr
    [npm]/[gatsby-cli]/[execa]/index.js:236:11

  - util.js:16 tryCatcher
    [npm]/[gatsby-cli]/[bluebird]/js/release/util.js:16:23

  - promise.js:512 Promise._settlePromiseFromHandler
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:512:31

  - promise.js:569 Promise._settlePromise
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:569:18

  - promise.js:614 Promise._settlePromise0
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18

  - promise.js:638 Promise._fulfill
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:638:18

  - promise_array.js:126 PromiseArray._resolve
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:126:19

  - promise_array.js:144 PromiseArray._promiseFulfilled
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:144:14

  - promise.js:574 Promise._settlePromise
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:574:26

  - promise.js:614 Promise._settlePromise0
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18

  - async.js:138 _drainQueueStep
    [npm]/[gatsby-cli]/[bluebird]/js/release/async.js:138:12

  - async.js:131 _drainQueue
    [npm]/[gatsby-cli]/[bluebird]/js/release/async.js:131:9

  - async.js:147 Async._drainQueues
    [npm]/[gatsby-cli]/[bluebird]/js/release/async.js:147:5

  - async.js:17 Immediate.Async.drainQueues [as _onImmediate]
    [npm]/[gatsby-cli]/[bluebird]/js/release/async.js:17:14



E:\Computer-Stuff\Javascript\2018>
```

When running gatsby develop:

```

E:\Computer-Stuff\Javascript\2018\gatsby-site>gatsby develop
success open and validate gatsby-config — 0.007 s
success load plugins — 0.112 s
error Plugin gatsby-plugin-manifest returned an error


  Error: Cannot find module '../build/Release/sharp.node'

  - loader.js:581 Function.Module._resolveFilename
    internal/modules/cjs/loader.js:581:15

  - loader.js:507 Function.Module._load
    internal/modules/cjs/loader.js:507:25

  - loader.js:637 Module.require
    internal/modules/cjs/loader.js:637:17

  - v8-compile-cache.js:159 require
    [gatsby-site]/[v8-compile-cache]/v8-compile-cache.js:159:20

  - constructor.js:10 Object.<anonymous>
    [gatsby-site]/[sharp]/lib/constructor.js:10:15

  - v8-compile-cache.js:178 Module._compile
    [gatsby-site]/[v8-compile-cache]/v8-compile-cache.js:178:30

  - loader.js:700 Object.Module._extensions..js
    internal/modules/cjs/loader.js:700:10

  - loader.js:599 Module.load
    internal/modules/cjs/loader.js:599:32

  - loader.js:538 tryModuleLoad
    internal/modules/cjs/loader.js:538:12

  - loader.js:530 Function.Module._load
    internal/modules/cjs/loader.js:530:3

  - loader.js:637 Module.require
    internal/modules/cjs/loader.js:637:17

  - v8-compile-cache.js:159 require
    [gatsby-site]/[v8-compile-cache]/v8-compile-cache.js:159:20

  - index.js:3 Object.<anonymous>
    [gatsby-site]/[sharp]/lib/index.js:3:15

  - v8-compile-cache.js:178 Module._compile
    [gatsby-site]/[v8-compile-cache]/v8-compile-cache.js:178:30

  - loader.js:700 Object.Module._extensions..js
    internal/modules/cjs/loader.js:700:10

  - loader.js:599 Module.load
    internal/modules/cjs/loader.js:599:32

  - loader.js:538 tryModuleLoad
    internal/modules/cjs/loader.js:538:12

  - loader.js:530 Function.Module._load
    internal/modules/cjs/loader.js:530:3

  - loader.js:637 Module.require
    internal/modules/cjs/loader.js:637:17

  - v8-compile-cache.js:159 require
    [gatsby-site]/[v8-compile-cache]/v8-compile-cache.js:159:20

  - gatsby-node.js:13 Object.<anonymous>
    [gatsby-site]/[gatsby-plugin-manifest]/gatsby-node.js:13:13

  - v8-compile-cache.js:178 Module._compile
    [gatsby-site]/[v8-compile-cache]/v8-compile-cache.js:178:30


error UNHANDLED REJECTION


  TypeError: Cannot read property 'filter' of undefined

  - api-runner-node.js:274 Promise.mapSeries.catch.then.results
    [gatsby-site]/[gatsby]/dist/utils/api-runner-node.js:274:42

  - util.js:16 tryCatcher
    [gatsby-site]/[bluebird]/js/release/util.js:16:23

  - promise.js:512 Promise._settlePromiseFromHandler
    [gatsby-site]/[bluebird]/js/release/promise.js:512:31

  - promise.js:569 Promise._settlePromise
    [gatsby-site]/[bluebird]/js/release/promise.js:569:18

  - promise.js:614 Promise._settlePromise0
    [gatsby-site]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [gatsby-site]/[bluebird]/js/release/promise.js:694:18

  - async.js:138 _drainQueueStep
    [gatsby-site]/[bluebird]/js/release/async.js:138:12

  - async.js:131 _drainQueue
    [gatsby-site]/[bluebird]/js/release/async.js:131:9

  - async.js:147 Async._drainQueues
    [gatsby-site]/[bluebird]/js/release/async.js:147:5

  - async.js:17 Immediate.Async.drainQueues [as _onImmediate]
    [gatsby-site]/[bluebird]/js/release/async.js:17:14



E:\Computer-Stuff\Javascript\2018\gatsby-site>
```

When running `npm install`

```
E:\Computer-Stuff\Javascript\2018\gatsby-site>npm install
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\browserslist.cmd as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\browserslist
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\browserslist as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\browserslist
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\regjsparser.cmd as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\regjsparser
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\regjsparser as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\regjsparser
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\jsesc.cmd as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\jsesc
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\jsesc as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\jsesc
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\detect.cmd as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\detect-port
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\detect-port.cmd as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\detect-port
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\detect as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\detect-port
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\detect-port as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\detect-port
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\babylon.cmd as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\babylon
npm WARN rm not removing E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\.bin\babylon as it wasn't installed by E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\babylon

> sharp@0.20.8 install E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp
> (node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)

info sharp Using cached C:\Users\Super\AppData\Roaming\npm-cache\_libvips\libvips-8.6.1-win32-x64.tar.gz
(node:24924) UnhandledPromiseRejectionWarning: ZlibError: zlib: unexpected end of file
    at Zlib.(anonymous function).onerror (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:131:21)
    at Unzip.write (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:255:31)
    at Unzip.flush (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:222:10)
    at Unzip.end (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minizlib\index.js:229:10)
    at Unpack.end (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\tar\lib\parse.js:415:21)
    at Yallist.pipes.forEach.p (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minipass\index.js:288:18)
    at Yallist.forEach (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minipass\node_modules\yallist\yallist.js:154:8)
    at ReadStream.emit (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minipass\index.js:285:18)
    at ReadStream.emit (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\fs-minipass\index.js:153:22)
    at ReadStream.[maybeEmitEnd] (E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\minipass\index.js:255:12)
(node:24924) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:24924) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
info sharp Creating E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\build\Release
info sharp Copying DLLs from E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\vendor\lib to E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\build\Release
prebuild-install WARN install The specified module could not be found.
prebuild-install WARN install \\?\E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp\build\Release\sharp.node

E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (node "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" rebuild )
gyp ERR! configure error
gyp ERR! stack Error: Command failed: c:\users\super\appdata\local\programs\python\python36-32\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                                ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:289:12)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:962:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
gyp ERR! System Windows_NT 10.0.17134
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd E:\Computer-Stuff\Javascript\2018\gatsby-site\node_modules\sharp
gyp ERR! node -v v10.10.0
gyp ERR! node-gyp -v v3.8.0
gyp ERR! not ok
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.4 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! sharp@0.20.8 install: `(node install/libvips && node install/dll-copy && prebuild-install) || (node-gyp rebuild && node install/dll-copy)`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the sharp@0.20.8 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Super\AppData\Roaming\npm-cache\_logs\2018-09-19T16_39_54_349Z-debug.log

E:\Computer-Stuff\Javascript\2018\gatsby-site>
```

### Environment

Run `gatsby info --clipboard` in your project directory and paste the output here. Not working? You may need to update your global gatsby-cli - `npm install -g gatsby-cli`
```
E:\Computer-Stuff\Javascript\2018>gatsby info --clipboard

  System:
    OS: Windows 10
    CPU: x64 Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz
  Binaries:
    Yarn: 1.9.4 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.4.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: 42.17134.1.0

error The system cannot find the path specified.



  Error: The system cannot find the path specified.

  - envinfo.js:1 Function.e.exports.sync
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:7778

  - envinfo.js:1 Object.copySync
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:104976

  - envinfo.js:1 Object.t.writeSync.e [as writeSync]
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:123499

  - envinfo.js:1
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:124274

  - envinfo.js:1 Promise.all.then.e
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:124289

  - util.js:16 tryCatcher
    [npm]/[gatsby-cli]/[bluebird]/js/release/util.js:16:23

  - promise.js:512 Promise._settlePromiseFromHandler
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:512:31

  - promise.js:569 Promise._settlePromise
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:569:18

  - promise.js:614 Promise._settlePromise0
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18

  - promise.js:638 Promise._fulfill
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:638:18

  - promise_array.js:126 PromiseArray._resolve
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:126:19

  - promise_array.js:144 PromiseArray._promiseFulfilled
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:144:14

  - promise.js:574 Promise._settlePromise
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:574:26

  - promise.js:614 Promise._settlePromise0
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18


error UNHANDLED REJECTION


  Error: The system cannot find the path specified.

  - envinfo.js:1 Function.e.exports.sync
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:7778

  - envinfo.js:1 Object.copySync
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:104976

  - envinfo.js:1 Object.t.writeSync.e [as writeSync]
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:123499

  - envinfo.js:1
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:124274

  - envinfo.js:1 Promise.all.then.e
    [npm]/[gatsby-cli]/[envinfo]/dist/envinfo.js:1:124289

  - util.js:16 tryCatcher
    [npm]/[gatsby-cli]/[bluebird]/js/release/util.js:16:23

  - promise.js:512 Promise._settlePromiseFromHandler
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:512:31

  - promise.js:569 Promise._settlePromise
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:569:18

  - promise.js:614 Promise._settlePromise0
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18

  - promise.js:638 Promise._fulfill
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:638:18

  - promise_array.js:126 PromiseArray._resolve
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:126:19

  - promise_array.js:144 PromiseArray._promiseFulfilled
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:144:14

  - promise.js:574 Promise._settlePromise
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:574:26

  - promise.js:614 Promise._settlePromise0
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [npm]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18



E:\Computer-Stuff\Javascript\2018>
```

**Comment by kakadiadarpan** [comment]
I'm unable to reproduce this error. Might be a `Windows` only issue?

**Comment by dumblole** [comment]
How might I fix this?

**Comment by kakadiadarpan** [comment]
Can you try uninstalling `gatsby-cli` and then reinstalling it?

**Comment by dumblole** [comment]
@kakadiadarpan I tried `npm uninstall -g gatsby-cli` and then did `npm install -g gatsby-cli` but with no success

**Comment by codejet** [comment]
@kakadiadarpan I'm having a similar issue on Mac when doing `gatsby new gatsby-blog https://github.com/gatsbyjs/gatsby-starter-blog`

I'm using gatsby-cli 2.4.1 and Node 8.1.4 on MacOS 10.13.6.

```
info Creating new site from git: https://github.com/gatsbyjs/gatsby-starter-blog.git
Cloning into 'gatsby-blog'...
remote: Enumerating objects: 1, done.
remote: Counting objects: 100% (1/1), done.
remote: Total 1139 (delta 0), reused 0 (delta 0), pack-reused 1138
Receiving objects: 100% (1139/1139), 2.14 MiB | 2.51 MiB/s, done.
Resolving deltas: 100% (636/636), done.
success Created starter directory layout
info Installing packages...
yarn install v1.9.4
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] 🔍  Resolving packages...
warning gatsby-plugin-sharp > imagemin-webp > cwebp-bin > bin-build > download > gulp-decompress > gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
[2/4] 🚚  Fetching packages...
error css-loader@1.0.0: The engine "node" is incompatible with this module. Expected version ">= 6.9.0 <7.0.0 || >= 8.9.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
error Command failed: yarnpkg


  Error: Command failed: yarnpkg

  - index.js:236 Promise.all.then.arr
    [lib]/[gatsby-cli]/[execa]/index.js:236:11

  - util.js:16 tryCatcher
    [lib]/[gatsby-cli]/[bluebird]/js/release/util.js:16:23

  - promise.js:512 Promise._settlePromiseFromHandler
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:512:31

  - promise.js:569 Promise._settlePromise
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:569:18

  - promise.js:614 Promise._settlePromise0
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18

  - promise.js:638 Promise._fulfill
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:638:18

  - promise_array.js:126 PromiseArray._resolve
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:126:19

  - promise_array.js:144 PromiseArray._promiseFulfilled
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise_array.js:144:14

  - promise.js:574 Promise._settlePromise
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:574:26

  - promise.js:614 Promise._settlePromise0
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:614:10

  - promise.js:694 Promise._settlePromises
    [lib]/[gatsby-cli]/[bluebird]/js/release/promise.js:694:18

  - async.js:138 _drainQueueStep
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:138:12

  - async.js:131 _drainQueue
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:131:9

  - async.js:147 Async._drainQueues
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:147:5

  - async.js:17 Immediate.Async.drainQueues
    [lib]/[gatsby-cli]/[bluebird]/js/release/async.js:17:14
```

When instead of doing this I clone the repo, cd into the directory and run `npm install` everything seems to work fine.

**Comment by ouzhou** [comment]
same issue both macOs and Windows
:(

**Comment by codejet** [comment]
I had an issue on macOS which sounded very similar, and it was due to having the wrong version of Node installed. But in my case it was pretty clear from the error message.

**Comment by dumblole** [comment]
@ouzhou  what error did you get? What does it say? Is it something to do with Sharp?

**Comment by DSchau** [comment]
@dumblole you solved this here, right? lovell/sharp#1388

I'm going to close this out as answered and outside of Gatsby's purview.

@ouzhou if you can provide more detail, please let us know! But definitely take a look at the solution I linked above ☝️

**Comment by dumblole** [comment]
Yes @DSchau
